sdk/src/assertions/actions.rs

```diff
@@ -119,8 +122,9 @@ pub struct Action {
 
     /// This is NOT the instanceID in the spec
     /// It is now deprecated but was previously used to map the action to an ingredient
-    #[serde(rename = "instanceId", skip_serializing)] // this should never be written to CBOR
     #[deprecated(since = "0.37.0", note = "Use `org.cai.ingredientIds` instead")]
+    #[serde(skip_serializing)]
+    #[serde(alias = "instanceId", alias = "instanceID")]
```

Comment:
There's one possible side effect to this that could be an issue. We also use serializing when I save a Builder archive. In that case the instanceId field would get lost. But it is deprecated now in favor of using an ingredient_ids field. So maybe that is ok.